dependency.lic

```diff
@@ -552,6 +552,12 @@ class ScriptManager
     message_body += "\n"
     message_body += "Dependency Version: #{$DEPENDENCY_VERSION}"
     message_body += "\n"
+    
+    message_body += "\n****\n" * 3
+	overrideFile = File.path('./scripts/profiles/override.yaml')
+	message_body += overrideFile.to_s
+    message_body += "\n"
+    message_body += File.read(overrideFile)
```

Comment:
Not sure about the override naming convention for this, it's not an end-all, I just find it possibly misleading. 

Possible alternatives:

1. common.yaml
2. base-common.yaml
3. common-settings.yaml

Just some examples.